Markdown/Code Block Syntaxes.sublime-completions

```diff
@@ -0,0 +1,362 @@
+{
+	"scope": "text.html.markdown meta.code-fence.definition.begin",
+	"completions": [
+		{
+			"trigger": "bash",
+			"contents": "bash",
```

Comment:
`contents` can be omitted if they are the same as the trigger.

Reply:
I just followed the scheme of other Default Packages, whose completions were converted to support kind info. All of them explicitly contain `contents`.